en/controllers/middleware.rst

```diff
@@ -45,11 +70,11 @@ hook method will be called early in the request process, you can use the
 
     class Application extends BaseApplication
     {
-        public function middleware($middlewareStack)
+        public function middleware($middleware)
```

Comment:
Oh, thanks for pointing that out. But then it should be named ``middlewareQueue``, as in https://github.com/cakephp/app/blob/6d9a9909fea0f06fe305d6095c1b33d3cac65e80/src/Application.php#L39. Otherwise we need to update the docs later again.